kernel/src/fs/ext2/inode.rs

```diff
@@ -2173,7 +2175,7 @@ impl TryFrom<RawInode> for InodeDesc {
             mtime: Duration::from(inode.mtime),
             dtime: Duration::from(inode.dtime),
             hard_links: inode.hard_links,
-            blocks_count: inode.blocks_count,
+            blocks_count,
```

Comment:
Let's rename this field to `sector_count` (not in the plural form) to avoid making the same mistake in the future. And for consistency, could you also rename `blocks_count` to `block_count` (in a separate commit)?